### PR TITLE
Menu improvements

### PR DESCRIPTION
**Pull request by Benibur**
A collection of UI/UX improvements :
* prevent the menu to open when clicking in a form
* improved material inputs responsiveness of the in page login menu
* new icon for inPageMenu in host page fields

### CODE REVIEW

File: src/content/menuCtrler.js [hunk 1]
```diff
@@ -35,14 +29,14 @@ var menuCtrler = {
...
     activate               : null,
-    // displayLoginIPMenu : null,  // to be removed ? BJA
 }
 
 /* --------------------------------------------------------------------- */
 // GLOBALS
 var menuEl,
     popperInstance,
-    targetsEl = [],
+    targetsEl = [],  // fields where
```
[the remark this answers is not in the document]
Comment (Benibur) [reply src/content/menuCtrler.js:38]: good point, taken into account !

**Review by ptbrowne — APPROVED** [review]

File: src/content/menuCtrler.js [hunk 2]
```diff
@@ -93,8 +93,9 @@ function _initInPageMenuForEl(targetEl) {
     targetEl.style.backgroundImage = menuBtnSvg
     targetEl.style.backgroundRepeat = "no-repeat"
     targetEl.style.backgroundAttachment = "scroll"
-    targetEl.style.backgroundSize = "24px 24px"
-    targetEl.style.backgroundPosition = "calc(100% - 16px) 50%"
+    targetEl.style.backgroundSize = targetEl.clientWidth > 90 ? "24px 24px" : "10px 10px"
+    const targetWidth = targetEl.clientWidth
```
Comment (ptbrowne) [inline src/content/menuCtrler.js:97]: This variable is created but is not used.

**Comment by ptbrowne** [comment]
Let's merge this PR after the fix of the targetWidth variable ?

File: src/inPageMenu/menu.js [hunk 3]
```diff
@@ -3,10 +3,13 @@ require('./menu.scss');
...
 var ciphers                 ,
     panel                   ,
-    resizeListener   = null ,
+    arrow                   ,
+    currentArrowD  = null   ,
+    resizeListener = null   ,
     lastSentHeight          ,
     titleEl                 ,
     i18nGetMessage          ,
+    currentArrowD           ,
```
Comment (ptbrowne) [inline src/inPageMenu/menu.js:12]: Declared twice, currentArrowOffset could be a better name.

**Review by ptbrowne — APPROVED** [review]

File: src/content/menuCtrler.js (hunk 2 above)
Comment (Benibur) [reply src/content/menuCtrler.js:100]: +1 : corrected

File: src/inPageMenu/menu.js (hunk 3 above)
Comment (Benibur) [reply src/inPageMenu/menu.js:12]: relevant, var renamed

**Pull request merged**